omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -759,7 +759,7 @@ def apply_rdefs(self, image, channels):
                 c_idxs.append(i+1)
                 windows.append([c['window']['start'], c['window']['end']])
                 colors.append(c['color'])
-                reverses.append(c['reverseIntensity'])
+                reverses.append(c.get('reverseIntensity', False))
```

Comment:
we will also need old testing data in our setup.
I have created https://trello.com/c/h9WOaIkv/10-produce-older-figures-in-the-testing-db
unless we run nightshade clone somewhere later on on